firestarter/cli/parser.py

```diff
@@ -27,15 +27,28 @@ def main():
     arg_parser.add_argument('--config_file', type=str, help='Optional configuration file for the workflow, located in the repository')
     args = arg_parser.parse_args()
 
+    vars = {}
+    secrets = {}
+    config_file = None
+
+    input_vars = os.environ.get("INPUT_VARS", None)
+    input_secrets = os.environ.get("INPUT_SECRETS", None)
+    input_config_file = os.environ.get("INPUT_CONFIG_FILE", None)
+
+    if None not in (input_vars, input_secrets, input_config_file):
+      vars.update(tomllib.loads(input_vars))
+      secrets.update(tomllib.loads(input_secrets))
+      config_file = input_config_file if input_config_file is not None else args.config_file
```

Comment:
I would use an if for every input, if not this if is going to be skipped with any of the inputs being None